### PR TITLE
Added first version of ihp-troubleshoot command

### PR DESCRIPTION
**Pull request by mpscholten**
Fixes https://github.com/digitallyinduced/ihp/issues/413

### CODE REVIEW

**Comment by HugoPeters1024** [comment]
This is a cool idea!

File: Troubleshoot/ihp-troubleshoot [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+step "GHCI Output"
+echo "$ghci_output"
+step "build/ihp-lib Target"
+echo "$ihplib_link"
+step "Direnv Output"
+echo "$direnv_output"
+step "which direnv Output"
+echo "$which_direnv_output"
+step "default.nix"
+echo "$(cat default.nix|head -n5)"
```
Comment (hendi) [inline Troubleshoot/ihp-troubleshoot:118]: No need for the `echo`, just do `cat default.nix|head -n5`

Comment (hendi) [reply Troubleshoot/ihp-troubleshoot:118]: Also, no need for `cat`, just do `head -n5 default.nix` :wink:

Comment (mpscholten) [reply Troubleshoot/ihp-troubleshoot]: Good catch :) Just removed it 👍

**Pull request merged**